codespell_lib/_codespell.py

```diff
@@ -54,9 +56,9 @@
     ('usage', 'for recommended terms', '_usage',
         None, None, None, None),
     ('code', 'for words common to code and/or mathematics', '_code',
-        None, None, None, None,),  # noqa: E501
+        None, None, None, None,),
     ('names', 'for valid proper names that might be typos', '_names',
-        None, None, None, None,),  # noqa: E501
+        None, None, None, None,),
     ('en-GB_to_en-US', 'for corrections from en-GB to en-US', '_en-GB_to_en-US',  # noqa: E501
```

Comment:
Looks like this one just sneaks under the line too now.

Reply:
No my flake is comlaining

Reply:
Okay.